src/history.py

```diff
@@ -33,134 +26,86 @@ def load_df(root=None, cleaned=False, end=False) -> pd.DataFrame:
 
     # get cleaned if current df is dirty
     if not end:
+        # TODO clean this 
```

Comment:
Keeping this here for now, since it has to do with data updates. Will need to add testing around this behavior